content/en/docs/collector/transforming-telemetry.md

```diff
@@ -169,6 +169,31 @@ processors:
   k8sattributes/default:
 ```
 
+## Setting a span status
+
+**Processor**:
+[transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor)
+
+Use the transform processor to set a span's status. The following example sets
+the span status to `Ok` when the `http.status_code` attribute is 400:
```

Comment:
```suggestion
the span status to `Ok` when the `http.request.status_code` attribute is 200:
```
I understand it's just an example, but would using an actual HTTP success status code make more sense? It jumped out to me that we're setting the span status to `Ok` when we see a failure status code.

Also, we need to make sure the attribute here lines up with the example.

Reply:
This is intentional, the 400 has probably set `Error` as status code in the SDK code, and then in the collector we want to revert that, explicitly stating that those errors are OK for us.

I agree with the missing `request`

Reply:
>This is intentional, the 400 has probably set `Error` as status code in the SDK code, and then in the collector we want to revert that, explicitly stating that those errors are OK for us.

Got it, thanks for the explanation. Could we include a short description of our goal here, then? My first impression was that the span was starting with an `Unknown` status code or similar and we wanted to populate it with the correct information.

Reply:
Yes, it would make sense to be a little bit clearer here.

Reply:
```suggestion
the span status to `Ok` when the `http.request.status_code` attribute is 200:
```

@Gaurang-Patel please apply this change

Reply:
Hi @svrnm , it will be like 
```the span status to `Ok` when the `http.request.status_code` attribute is 400:```
 correct? as in example the condition is on 400.

Reply:
exactly! All that needs to be added is that "request"